src/components/topbar/topbar.css

```diff
@@ -69,6 +69,34 @@
     .topbar .hamburger {
         margin-right: 16px;
     }
+    .topbar .right-list{
+        display: none;
+    }
+}
+
+.popContent {
+    padding: 10px 30px;
+}
+
+.topbar .ant-menu-horizontal > .ant-menu-item,
+.ant-menu-horizontal > .ant-menu-item:hover, 
+.ant-menu-horizontal > .ant-menu-submenu:hover, 
+.ant-menu-horizontal > .ant-menu-item-active, 
+.ant-menu-horizontal > .ant-menu-submenu-active, 
+.ant-menu-horizontal > .ant-menu-item-open, 
+.ant-menu-horizontal > .ant-menu-submenu-open, 
+.ant-menu-horizontal > .ant-menu-item-selected, 
+.ant-menu-horizontal > .ant-menu-submenu-selected {
+    border: none;
+}
+
+.ant-menu-horizontal{
+    border: none;
+    background-color: initial;
+}
+
+.topbar .ant-popover{
```

Comment:
This is not working as `.ant-popover` is coming outside of `.topbar`. Remove this style from here and instead use `overlayStyle` props on the `Popover` component to center align content